### PR TITLE
Week2

### PR DESCRIPTION
**Pull request by maccrin**
This homework on meal sharing wekk2

### CODE REVIEW

File: src/backend/api/meals.js [hunk 1]
```diff
@@ -5,11 +5,69 @@ const knex = require("../database");
...
+  catch (error) {
+    response.status(503).send('Not able to create new meal')
+  }
+});
+router.put("/:id", async (req, res) => {
+  const mealId = parseInt(req.params.id);
+  try {
+    const updatedMeal = await knex('meal')
+      .where({ id: mealId })
+      .update({ description: req.body.description })
```
Comment (Divyajg) [inline src/backend/api/meals.js]: It should be possible to update more than one field. I suggest you to modify this.

File: src/backend/api/reservations.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    catch (error) {
+        response.status(503).send('Not able to create new reservation')
+    }
+});
+router.put("/:id", async (req, res) => {
+    const reserveId = parseInt(req.params.id);
+    try {
+        const updatedReservation = await knex('reservation')
+            .where({ id: reserveId })
+            .update({ contact_name: req.body.contact_name })
```
Comment (Divyajg) [inline src/backend/api/reservations.js]: Same as in meals.

**Review by Divyajg — COMMENTED** [review]
Very good work.

File: src/backend/api/meals.js (hunk 1 above)
Comment (maccrin) [reply src/backend/api/meals.js]: I will do  it

File: src/backend/api/reservations.js (hunk 2 above)
Comment (maccrin) [reply src/backend/api/reservations.js]: 👍🏻

**Review by Divyajg — APPROVED** [review]

File: src/backend/api/reservations.js (hunk 2 above)
Comment (maccrin) [reply src/backend/api/reservations.js]: Thank You for your time 🙏🏾

**Pull request merged**